base/ltoutput.dtx

```diff
@@ -2350,9 +2402,18 @@
 %    Then the footnotes are appended:
 %    \begin{macrocode}
      \@outputbox@append{%
+%    \end{macrocode}
+%    First we call a socket which may or may not back up by the depth
+%    of the \cs{@outputbox} so that we start adding from the baseline
+%    of the last line.
+%    Then we add the skip above the footnotes.
```

Comment:
Does this make sense in context? 
What skip is "the skip" that we add? 
Maybe start with: 
  Following this we also add . . .

Reply:
rewritten